EmbyStat.Jobs/Jobs/Sync/MediaSyncJob.cs

```diff
@@ -193,9 +193,15 @@ private async Task PerformShowSyncAsync(IReadOnlyList<string> showsThatNeedAnUpd
                 {
                     var seasons = _httpClient.GetSeasons(showDto.Id);
                     var episodes = _httpClient.GetEpisodes(seasons.Select(x => x.Id), show.Id);
+                    double size = 0;
 
                     show.Seasons.AddRange(seasons.Where(x => show.Seasons.All(y => y.Id != x.Id)));
                     show.Episodes.AddRange(episodes.Where(x => show.Episodes.All(y => y.Id != x.Id)));
+
+                    size = show.GetShowSize();
```

Comment:
I like the size prop so I will add it to the database. 
One tip, you don't need to have a local "size" object. You can just do show.Size = show.GetShowSize(); and in the log show.Size